internal/app/feed/export_feeds.go

```diff
@@ -9,24 +9,27 @@ import (
 	"github.com/spf13/viper"
 )
 
+var feeds = []Feed{
+	&InspectionFeed{},
```

Comment:
We should reuse this for ExportFeeds as well. You can fetch the viper config inside `GetFeeds`